package_manifest.yaml

```diff
@@ -7,12 +7,17 @@ packages:
     katello_client_packages: {}
 
 foreman_packages:
+  children:
+    foreman_scl_packages: {}
+    foreman_nonscl_packages: {}
+    foreman_scl_nonscl_packages: {}
```

Comment:
@ekohl Can you re-review, as I made this change here. In the current form, I think this is required to handle dual packages. Even then, diff-package does not work properly due to how package names are derived and them living a double life (goes back to the discussion @evgeni started our SCL and non-SCL naming). I think I will need to modify diff package to accept some special arguments per tag since I doubt we will move forward the naming discussion in time

Reply:
Note @ekohl this really is just whether you agree with this dual group here. The diff package issues can be fixed afterward as they exist no matter what breakdown we choose here.